src/Raven.Studio/typescript/models/database/tasks/sql/sqlMigration.ts

```diff
@@ -111,6 +129,8 @@ class sqlMigration {
                 return "Microsoft SQL Server (System.Data.SqlClient)";
             case "MySQL":
                 return "MySQL Server (MySql.Data.MySqlClient)";
+            case "NpgSQL":
+                return "PostgreSQL Server";
```

Comment:
Please follow convention for different driver name (Driver Class Name), see 2 and 4 lines above.

Reply:
done